src/components/ContentNode/SectionTitle.vue

```diff
@@ -0,0 +1,86 @@
+<!--
+  This source file is part of the Swift.org open source project
+
+  Copyright (c) 2022 Apple Inc. and the Swift project authors
+  Licensed under Apache License v2.0 with Runtime Library Exception
+
+  See https://swift.org/LICENSE.txt for license information
+  See https://swift.org/CONTRIBUTORS.txt for Swift project authors
+-->
+
+<template>
+  <component
+   :id="!isTargetIDE ? id : null"
+   :is="tag"
+   class="section-title"
+  >
+    <a
+      v-if="anchor && !isTargetIDE"
+      :href="`#${anchor}`"
+      class="header-anchor"
+      aria-label="hidden"
+      @click="handleFocusAndScroll(anchor)"
+    >#</a>
+    <slot />
+  </component>
+</template>
+
+<script>
+import scrollToElement from 'docc-render/mixins/scrollToElement';
+
+export default {
+  name: 'SectionTitle',
+  mixins: [scrollToElement],
+  data() {
+    return {
+      id: null,
+    };
+  },
+  props: {
+    anchor: {
+      type: String,
+      required: false,
+    },
+    tag: {
+      type: String,
+      default: () => 'h2',
```

Comment:
Whether this stays as a `tag` or `level` prop, we should probably add a validator to ensure that you can only ever use a h1-h6 (or only a 1-6 level).

Reply:
Added!